### PR TITLE
Feature#18 add styles to feature main view

### LINKED ISSUE #18: Add styles to feature Main View
# REQUIREMENTS
In this issue all styles are going to be added in the Main View

# DESIGN
Provide references to images of the design, how it looks like now, and how it should look like.
image

# Acceptance Criteria
List all required acceptance criteria
- [x] The final result should be similar to the design

### PR DESCRIPTION
**Pull request by ArielMD**
I implemented styles to main view

I used bootstrap, sass and to  to make it similar to design and js to show the clock with the current time
 
## Description: Why did you change/implement that?

To give a better UI to the project

## Testing: Help me how can I test or look at the changes?

image
image

### CODE REVIEW

File: reloj-checador/app/assets/stylesheets/attendaces.scss [hunk 1]
```diff
@@ -1,3 +1,37 @@
...
+	}
+}
+
+.input-attendance{
+    .form-control{
+        border-top: none;
+        border-right: none;
+        border-left: none;
+    }
+}
```
Comment (YilKanda) [inline reloj-checador/app/assets/stylesheets/attendaces.scss:37]: Remember to have a blank space at the end of each file.

File: reloj-checador/app/views/attendaces/_body_table_attendaces.html.erb [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+  <tr>
+    <th scope="row"><%= attendace.id%></th>
+    <td><%= attendace.name%> </td>
+    <td><%= attendace.lastname%> </td> 
+    <td><%= attendace.position%> </td> 
+    <td><%= attendace.date%> </td> 
+    <td><%= attendace.time%> </td>
+    <td><%= attendace.check_type%> </td> 
+  </tr>
+<%end%>
```
Comment (YilKanda) [inline reloj-checador/app/views/attendaces/_body_table_attendaces.html.erb:11]: Remember to have a blank space at the end of each file.

File: reloj-checador/app/views/attendaces/_form.html.erb [hunk 3]
```diff
@@ -1,27 +1,27 @@
...
-<% end %>
+<script>
+  let clock = document.getElementById("clock")
+  
+  setInterval(() => {
+    let date = new Date()
+    clock.innerHTML = date.toLocaleTimeString();
+    console.log(date.toLocaleTimeString());
+  },1000);
+</script>
```
Comment (YilKanda) [inline reloj-checador/app/views/attendaces/_form.html.erb]: Remember to have a blank space at the end of each file.

**Review by YilKanda — COMMENTED** [review]
The line break at the end of some files is missing, to comply with the Ruby Style Guide. They also check for conflicts that exist between the branch and the master. Remember to rebase the origin/master.

`git fetch`

`git rebase origin/master`

File: reloj-checador/app/assets/stylesheets/attendaces.scss (hunk 1 above)
Comment (ArielMD) [reply reloj-checador/app/assets/stylesheets/attendaces.scss:37]: thank, I added line break  when I resolved the conflicts between the branches

**Pull request merged**